pkg/pillar/scripts/device-steps.sh

```diff
@@ -302,11 +341,16 @@ if [ -f /var/run/watchdog.pid ]; then
 fi
 /usr/sbin/watchdog -c $TMPDIR/watchdognim.conf -F -s &
 
+# Print diag output forever on changes
+$BINDIR/diag -c $CURPART -f >/dev/console 2>&1 &
+
 # Wait for having IP addresses for a few minutes
 # so that we are likely to have an address when we run ntp
 echo "$(date -Ins -u) Starting waitforaddr"
 $BINDIR/waitforaddr -c $CURPART
 
+access_usb
```

Comment:
Not clear to me why "access_usb" function should be called twice. One at line 316 and again at line 352. A comment might help.

Reply:
I can add that. Idea is to dump the diag output when onboarding has issues (with proxy or static IP) without requiring a console.